feat/gene/gene.go

```diff
@@ -285,6 +318,39 @@ func (t *CodingTranscript) SetExons(exons ...Exon) error {
 	return nil
 }
 
+// TranscriptFeature defines a feature on a transcript.
+type TranscriptFeature struct {
+	Transcript Transcript       // Transcript is the transcript that the feature is located.
+	Offset     int              // Offset is the position of the feature relative to Transcript.
```

Comment:
Sorry for the late reply. Just for consistency with the other types. Should I change it?